Classes/Auth/TMOAuth.m

```diff
@@ -75,6 +90,37 @@ - (id)initWithURL:(NSURL *)URL method:(NSString *)method postParameters:(NSDicti
 
 #pragma mark - Private
 
+NSMutableDictionary *generateOAuthParameters(TMOAuth *tmOAuth,
+                                             NSURL *URL,
+                                             NSString *method,
+                                             NSDictionary *postParameters,
+                                             NSString *nonce,
+                                             NSString *consumerKey,
+                                             NSString *consumerSecret,
+                                             NSString *token,
+                                             NSString *tokenSecret,
+                                             NSString *timestamp) {
+    NSMutableDictionary *oAuthParameters = [[NSMutableDictionary alloc] initWithDictionary:@{
+                                                                                              @"oauth_timestamp" : timestamp,
+                                                                                              @"oauth_nonce" : nonce,
+                                                                                              @"oauth_version" : @"1.0",
+                                                                                              @"oauth_signature_method" : @"HMAC-SHA1",
+                                                                                              @"oauth_consumer_key" : consumerKey,
+                                                                                              }];
+    if (token && token.length > 0)
```

Comment:
As mentioned above, we prefer including the curly braces in all control statements, even single-line ones (even though some existing code in this file isn't a great example of that).

Also, because it's acceptable to send a message to a `nil` object in Objective-C, it's more idiomatic to omit the `nil` check in this `if` condition.